modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc

```diff
@@ -0,0 +1,39 @@
+////
+This module included in the following assemblies:
+-distr_tracing_install/distributed-tracing-deploying-otel.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_monitoring_{context}"]
+= Sending metrics to the monitoring stack
+
+The metrics from the {OTELName} can be consumed by the in-cluster monitoring stack. Refer to
+xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[enable monitoring for user defined projects] how to
+configure monitoring stack.
+
+.Sample `+PodMonitor+` that configures monitoring stack to scrape OpenTelemetry collector Prometheus exporter, internal metrics endpoint and removes duplicated labels.
```

Comment:
rather than being one large code sample, it seems like this should be made into a procedure module that explains the fields and what is being configured using [callouts](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)

Reply:
I have added 2 callouts for the ports